core-java-modules/core-java-nio/pom.xml

```diff
@@ -0,0 +1,97 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<artifactId>core-java-nio</artifactId>
+	<version>0.1.0-SNAPSHOT</version>
+	<name>core-java-nio</name>
+	<packaging>jar</packaging>
+
+	<parent>
+		<groupId>com.baeldung</groupId>
+		<artifactId>parent-java</artifactId>
+		<version>0.0.1-SNAPSHOT</version>
+		<relativePath>../parent-java</relativePath>
+	</parent>
+
+	<dependencies>
+	</dependencies>
+
+	<build>
+		<finalName>core-java-nio</finalName>
+		<resources>
+			<resource>
+				<directory>src/main/resources</directory>
+				<filtering>true</filtering>
+			</resource>
+		</resources>
+		<plugins>
+			<plugin>
+				<groupId>org.codehaus.mojo</groupId>
+				<artifactId>exec-maven-plugin</artifactId>
```

Comment:
This looks to be trying to run a `com.baeldung.outofmemoryerror.OutOfMemoryGCLimitExceed` class... this plugin shouldn't be needed!

Reply:
Yes. removed.